README.md

```diff
@@ -1 +1,144 @@
-# Bookstore-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Math Magician Project
+
+# 📗 Table of Contents
+
+- [Math Magician Project](#math-magician-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Math Magician Project ](#-math-magician-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+    - [Run tests ](#run-tests-)
+  - [👥 Authors ](#-authors-)
+    - [👤 **Mert Kantaroglu**](#-mert-kantaroglu)
+    - [👤 **Josue Rushanika**](#-josue-rushanika)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Math Magician Project <a name="about-project"></a>
+
+**Math Magician is a SPA project which you can make single calculations and reading a random math related quote**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **Homepage**
+- **Calculator**
+- **Quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>Git</li>
+    <li>Jest</li>
+    <li>React</li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Portfolio presentation will be coming soon.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd <desired-folder>
+  git clone https://github.com/mertkantaroglu/Math-Magician.git
+  cd math-magicians
```

Comment:
- [ ] Let's provide the correct URL for cloning the repo. This is a books store app and not a maths magician.